src/components/GridItem/GridItem.tsx

```diff
@@ -0,0 +1,33 @@
+// Importing React and its awesome hooks
+// import { JSXElement } from '@babel/types';
+import React from 'react';
+// importing props from the components TypeScript interface
+import { GridItemProps } from './types';
+import styles from './GridItem.styles';
+
+const { DivStyled } = styles;
+
+// Normal boilerplate React functional component
+const GridItem: React.FC<GridItemProps> = ({
+    // deconstructing props and setting any default values, sorted alphabetically
+    children,
+    gridArea = [],
+    ...props
```

Comment:
```jsx
.item1 {
  grid-area: myArea;
}
.grid-container {
  display: grid;
  grid-template-areas: 'myArea myArea myArea myArea myArea';
}
```
Just a thought, maybe using this grid-area like above would make more sense for this component and would give more customization options 😄 
Would like to hear your thoughts on this. 🙌🏻

Reply:
This is much easier for users to understand rather than using numbers for gridArea 😅, but I think it makes it complex when adding a bigger grid or when customizing. 
I will give it a try.🙌

Reply:
Hmm, i always thought `grid-template-areas` gave better customization options maybe I am wrong. 🤔 
No need to add this change if you feel this won't be that helpful.